theme.json

```diff
@@ -401,6 +401,28 @@
 						}
 					}
 				}
+			},
+			"core/query-pagination": {
+				"elements": {
+					"link": {
+						"typography": {
+							"textDecoration": "none",
+							"textTransform": "uppercase"
+						}
+					}
+				},
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--league-gothic)",
+					"fontSize": "var(--wp--preset--font-size--medium)"
+				}
+			},
+			"core/post-featured-image": {
+				"border": {
+					"color": "var(--wp--preset--color--foreground)",
+					"radius": "0.25rem",
+					"style": "solid",
+					"width": "2px"
```

Comment:
This should be `1px` as per the design.

Reply:
fixed here 1d3ddd0de288fd4c58c0532d1ae2d8ee6af3a96f